features/step_definitions/helpers.rb

```diff
@@ -46,14 +46,14 @@ def initialize_story_params(project, user = User.find(:first))
   story
 end
 
-def initialize_task_params(project, story, user = User.find(:first))
+def initialize_task_params(project, story, user = User.first)
   params = HashWithIndifferentAccess.new
   params['type_id'] = Task.type
   if story
     params['fixed_version_id'] = story.fixed_version_id
     params['parent_id']        = story.id
   end
-  params['status_id'] = Status.find(:first).id
+  params['status_id'] = Status.first.id
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.